x-pack/test/functional/apps/dashboard/group2/dashboard_maps_by_value.ts

```diff
@@ -54,8 +54,6 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     if (saveToLibrary) {
       await testSubjects.click('importFileButton');
       await testSubjects.click('mapSaveButton');
-      await PageObjects.timeToVisualize.ensureSaveModalIsOpen;
```

Comment:
The `await PageObjects.timeToVisualize.ensureSaveModalIsOpen;` was not actually **calling** the function - plus, it was unnecessary, since `saveFromModal` calls this internally - so, I removed it.